tests/network/conftest.py

```diff
@@ -230,3 +220,21 @@ def cnao_deployment(hco_namespace):
 @pytest.fixture(scope="session")
 def ovn_kubernetes_cluster(admin_client):
     return get_cluster_cni_type(admin_client=admin_client) == "OVNKubernetes"
+
+
+@pytest.fixture(scope="session", autouse=True)
+def network_sanity(hosts_common_available_ports, junitxml_plugin):
```

Comment:
I would name it something like `multi_nic_cluster`.

This is the opposite from CNV decision to mark tests that should run on specific lanes.
I understood that there are two clusters type, regular and special and assume that this full into special cluster type.
You eliminated the skip but introduced new problem that you don't know if those tests run or not (it will be dynamic)

Network tests should be marked as `@special`.